public/app/features/dashboard/dashgrid/DashboardEmpty.tsx

```diff
@@ -29,7 +29,6 @@ export const DashboardEmpty = ({ dashboard, canCreate }: Props) => {
           <Button
             size="lg"
             icon="plus"
-            aria-label="Add new panel"
```

Comment:
🤔 I think this is a good opportunity to add a `data-testid` and create an e2e flow and test that uses this new page :)

Reply:
True, I created a separate issue for that because I'll be able to get to it later: https://github.com/grafana/grafana/issues/69637